postgrest/_sync/request_builder.py

```diff
@@ -214,7 +312,7 @@ def delete(
             count: The method to use to get the count of rows returned.
             returning: Either 'minimal' or 'representation'
         Returns:
-            :class:`SyncFilterRequestBuilder`
+            :class:`AsyncFilterRequestBuilder`
```

Comment:
```suggestion
            :class:`SyncFilterRequestBuilder`
```
this is an unasync bug, sorry for the inconvenience 😦